src/metabase/query_processor/streaming/xlsx.clj

```diff
@@ -11,11 +11,11 @@
             [metabase.util :as u]
             [metabase.util.date-2 :as u.date]
             [metabase.util.i18n :refer [tru]])
-  (:import java.io.OutputStream
-           [java.time LocalDate LocalDateTime LocalTime OffsetDateTime OffsetTime ZonedDateTime]
-           [org.apache.poi.ss.usermodel Cell DataFormat DateUtil Workbook]
-           org.apache.poi.ss.util.CellRangeAddress
-           [org.apache.poi.xssf.streaming SXSSFRow SXSSFSheet SXSSFWorkbook]))
+  (:import (java.io OutputStream)
+           (java.time LocalDate LocalDateTime LocalTime OffsetDateTime OffsetTime ZonedDateTime)
+           (org.apache.poi.ss.usermodel Cell DataFormat DateUtil Workbook)
+           (org.apache.poi.ss.util CellRangeAddress)
+           (org.apache.poi.xssf.streaming SXSSFRow SXSSFSheet SXSSFWorkbook)))
```

Comment:
Is this the new standard `:import` syntax? I had the same and the namespace checker made me create one like you are changing here.

Reply:
Oh, I think my editor plugin makes this change automatically when I run "clean namespaces". I'm not sure what the standard is. I'll revert this manually.

Reply:
I don't know if we really have a preference...may just be a quirk in the linter. Might be a good thing to bring up on Slack or at a backend sync 👍